hbw/app/javascript/packs/hbw/components/form/select_table.js.jsx

```diff
@@ -25,13 +25,31 @@ modulejs.define('HBWFormSelectTable',
         };
       }
 
-      getInputValues = (value) => {
+      initNullablaTable (value) {
         if (this.props.params.multi) {
           return JSON.parse(value) || [];
         } else {
-          return value || '';
+          return value || 'null';
```

Comment:
It looks weird to me. We submit string `'null'` when actually should submit nothing. Also, for the 'multi' mode, we can deselect all values. But in the 'single' mode, we should click on the special row. Not sure we should change it in this PR but it will be a good next step I think.

Reply:
Ok, I wil revert it